letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -458,7 +471,10 @@ def _create_vhost(self, path):
                 is_ssl = True
 
         filename = get_file_path(path)
-        is_enabled = self.is_site_enabled(filename)
+        if self.conf("handle-sites"):
+            is_enabled = self.is_site_enabled(filename)
+        else:
+            is_enabled = True
```

Comment:
If  `conf("handle-sites") == False` it does not mean that the site is enabled. If  `conf("handle-sites") == True`  it means that we are allowed to `a2enmode` a site, which we don't do here, here we only check if the site is enabled. Therefore, I think that `474-477` should be reverted to the original `is_enabled = self.is_site_enabled(filename)`. Unless I'm missing something.